src/Client/IO.hs

```diff
@@ -315,7 +312,7 @@ signPackageForConfiguredUser pkg = do
       signRes <- signWithTezosClient (Right $ getBytesToSign pkg) ccUserAlias
       case signRes of
         Left err -> pure $ Left $ toString err
-        Right signature' -> pure $ addSignature pkg (pk, signature')
+        Right signature' -> pure $ addSignature pkg (pk, TSignature signature')
 
 getBalance :: (HasTezosRpc m) => Address -> m Natural
```

Comment:
Have you checked whether balance/allowance getters work with V2? I assume that they shouldn't because the storage structure has changed

Reply:
The storage hasn't in fact changed, I was too lazy to do this now ))  Only views on storage (`StoreHasSubmap` instances) are different.

Reply:
:pogchamp: